### PR TITLE
DOP-3222 follow up: Add unit tests for button uri

### PR DESCRIPTION
**Pull request by seungpark**
### Stories/Links:

[DOP-3222](https://jira.mongodb.org/browse/DOP-3222)
**no behavior change**, added some comments and added unit tests to ensure Button components use Link when passed an href

### Current Behavior:

[server docs](https://www.mongodb.com/docs/manual/)

### Staging Links:

[server docs](https://docs-mongodbcom-integration.corp.mongodb.com/master/docs/seung.park/DOP-3222-follow-up/)

### Notes:
previous PR fixed the issue, this PR adds unit tests to the change

### CODE REVIEW

File: tests/unit/Button.test.js [hunk 1]
```diff
@@ -4,8 +4,24 @@ import Button from '../../src/components/Button';
...
+jest.mock('../../src/components/Link');
+
+describe('button component', () => {
+  beforeEach(() => {
+    Link.mockImplementation(jest.requireActual('../../src/components/Link').default);
+  });
+
+  it('renders correctly', () => {
+    const tree = render(<Button nodeData={mockData} />);
+    tree.debug();
```
Comment (rayangler) [inline tests/unit/Button.test.js:18]: Should we remove this debug statement?

**Review by rayangler — COMMENTED** [review]
I think the change looks good to me overall! Just wanted to flag the debug statement in the test.

Is it possible to update the PR title to be descriptive/reflective of the change? Something like "DOP-3222 Follow-up: Add unit tests for button uri"

File: tests/unit/Button.test.js (hunk 1 above)
Comment (seungpark) [reply tests/unit/Button.test.js:18]: good catch. removed

**Comment by seungpark** [comment]
> I think the change looks good to me overall! Just wanted to flag the debug statement in the test.
> 
> Is it possible to update the PR title to be descriptive/reflective of the change? Something like "[DOP-3222](https://jira.mongodb.org/browse/DOP-3222) Follow-up: Add unit tests for button uri"

thanks for the catch. updated title

**Review by rayangler — APPROVED** [review]
LGTM!

**Review by casthewiz — APPROVED** [review]

**Pull request merged**